### ISSUE TITLE
Olde version React 15 dependency

### ISSUE THREAD

**Issue by TonioloF** [zsusac/ReactHTMLTableToExcel#31]
I'ts important to update module to the new version of react 17. I'ts generating a peerDependencies conflict on npm update command ou npm upgrade command. Look:

 npm info react-html-table-to-excel peerDependencies
{ react: '^15.x.x' }

confict:

npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.x.x" from react-html-table-to-excel@2.0.0
npm ERR! node_modules/react-html-table-to-excel
npm ERR!   react-html-table-to-excel@"^2.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

**Comment by cmascardo5** [comment]
I agree @TonioloF the package seems to be dependent on react version 15; however, react has made it to version 16 and 17. @zsusac is it possible to update react-html-table-to-excel to use a new version of react?

**Comment by dqve** [comment]
> 

?

**Comment by tranviet2010** [comment]
...
